src/main/java/org/umssdiplo/automationv01/core/utils/PropertyAccessor.java

```diff
@@ -53,15 +50,19 @@ public String getBrowser() {
         return getDataProperty(BROWSER);
     }
 
-    public String getBaseUrl() {
-        return getDataProperty(BASE_URL);
-    }
-
     public String getUser() {
         return getDataProperty(USER);
     }
 
     public String getPassword() {
         return getDataProperty(PASSWORD);
     }
+
+    public String getSmartHouseBaseUrl() {
+        return getDataProperty(SMARTHOUSE_URL);
+    }
+
+    public String getSHDataProperty(String data) {
```

Comment:
I understood your purpose, but we need to use for especific cases the PropertyAccessor